src/modules/mysql.nix

```diff
@@ -120,19 +120,20 @@ in
       cfg.package
     ];
 
-    env.MYSQLDATA = config.env.DEVENV_STATE + "/mysql";
-    env.MYSQLSOCKET = config.env.DEVENV_STATE + "/mysql.sock";
+    env.MYSQL_HOME = config.env.DEVENV_STATE + "/mysql";
+    env.MYSQL_UNIX_PORT = config.env.DEVENV_STATE + "/mysql.sock";
+    env.MYSQL_TCP_PORT = mkIf (cfg.settings.mysqld ? port) (toString cfg.settings.mysqld.port);
```

Comment:
![image](https://user-images.githubusercontent.com/6224096/204060110-45c0e7e1-ebf6-4bb2-99ef-d678aa89522b.png)

with just
```
mysql.enable = true;
```

Reply:
Ah, oops. I have fixed this issue in the latest version :+1: